idaes/apps/caprese/examples/cstr_rodrigo/cstr_rodrigo_model2.py

```diff
@@ -0,0 +1,174 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES), and is copyright (c) 2018-2021
+# by the software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia University
+# Research Corporation, et al.  All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and
+# license information.
+#################################################################################
+"""
+Test for Cappresse's module for NMPC.

```

Comment:
Change to something like "Caprese example for NMPC with Rodrigo's CSTR model"

Reply:
Okay!